Salesforce/InRule.Salesforce.Service.json

```diff
@@ -224,8 +217,8 @@
       "inrule:sf:api:securityToken": "[parameters('sfSecurityToken')]",
       "inrule:sf:api:consumerKey": "[parameters('sfConsumerKey')]",
       "inrule:sf:api:consumerSecret": "[parameters('sfConsumerSecret')]",
-      "inrule:sf:ruleService:basicUserName": "[parameters('executionServiceBasicUsername')]",
-      "inrule:sf:ruleService:basicPwd": "[parameters('executionServiceBasicPassword')]"
+      "inrule:sf:ruleService:apiKey": "[parameters('executionServiceApiKey')]",
+      "inrule:sf:ruleService:authType": "apikey"
```

Comment:
I think we decided that we didn't need the authType setting?

Reply:
Yeah I forgot to remove it from here. Removed.